### ISSUE TITLE
Git 2.12.2(2)

### ISSUE THREAD

**Issue by FeodorFitsner** [appveyor/ci#1482]

**Comment by FeodorFitsner** [comment]
After upgrading Git to 2.12.2.2 had to deal with CA bundle issue. Error was:

```
git clone -q --branch=master https://github.com/appveyor-tests/cache.git C:\projects\cache
fatal: unable to access 'https://github.com/appveyor-tests/cache.git/': error setting certificate verify locations:
  CAfile: C:/Program Files/Git/mingw64/libexec/ssl/certs/ca-bundle.crt
  CApath: none
```

To fix that opened `%programdata%\Git\config` and added:

```
[http]
	sslCAInfo = C:/Program Files/Git/mingw64/ssl/certs/ca-bundle.crt
```

**Comment by asottile** [comment]
Hmm, I'm still getting the error (that I verified was fixed by the snapshot on my local machine): https://ci.appveyor.com/project/asottile/pre-commit/build/1.0.968/job/k06t92r3swkrmvxr

Is there a way I can verify this was pushed?

**Comment by FeodorFitsner** [comment]
Was the error related to Git and fixed in 2.12.2.2?

**Comment by asottile** [comment]
Yes, here's my paper trail:
- https://github.com/appveyor/ci/issues/1483
- https://github.com/git-for-windows/git/issues/1106
- https://github.com/git-for-windows/git/issues/1106#issuecomment-292744938
- https://github.com/git-for-windows/git/commit/7df5db8d31e7b384b14e488c76933d46a9e76161

**Comment by FeodorFitsner** [comment]
Didn't know about snapshots. Git 2.12.2.2 was installed from the official location: https://github.com/git-for-windows/git/releases/tag/v2.12.2.windows.2 Not sure if that fix made 2.12.2.2 - may be it's scheduled for later release. Try installing 2.12.2.2 locally and see if that works.

**Comment by asottile** [comment]
Ah, indeed it is not fixed with 2.12.2.2, I'll wait for the next release then, thanks!
